frontend/src/app/actions/browser.js

```diff
@@ -0,0 +1,3 @@
+import { createActions } from 'redux-actions';
+
+export default createActions({}, 'setState');
```

Comment:
First parameter is for custom action creators - of which there are none, here - the rest is a shortcut list of identity functions. [Like here in `actions/addon.js`](https://github.com/mozilla/testpilot/blob/master/frontend/src/app/actions/addon.js#L3) where we have one custom action creator mapping the input to a payload. The identity functions are all just wrapping the payload in the [Flux Standard Action format](https://github.com/acdlite/flux-standard-action#example).

Doesn't *really* need to be exported, because we'll probably never use it in components & etc. But every other action module exports its actions, so just keeping with the pattern.